tests/conftest.py

```diff
@@ -86,3 +87,12 @@ def datasets():
         examples.load_airplane(),  # PolyData
         examples.load_structured(),  # StructuredGrid
     ]
+
+
+@fixture()
+def check_vtk_error(caplog):
+    """Ensure that no VTK error is thrown."""
+    yield
+    for record in caplog.get_records("call"):
+        if record.levelno == logging.ERROR and "vtk" in record.message:
+            fail("VTK error recorded")
```

Comment:
Would you be amenable to this?
```
            fail(f"VTK error recorded:\n\t{record.getMessage()}")
```
This makes the error message easier to find when I run tests in my IDE.

Reply:
Good idea, we should also find all errors and display all of them.

Reply:
With the new commit, it fails like this:

```txt
       if len(errors) > 0:
>           fail(f"VTK error recorded\n\t{errors}")
E           Failed: VTK error recorded
E           	['The update extent specified in the information for output port 0 on algorithm vtkTrivialProducer(0x19943b0) is 0 39 0 39 0 39, which is outside the whole extent 0 3 0 3 0 3.']
```